CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## [v2.30.0 _(Nov 21, 2022)_](https://github.com/omise/omise-magento/releases/tag/v2.30.0)
+- Add support omise refund after offline refund is triggered.
```

Comment:
Let's mention this fix as well.

https://github.com/omise/omise-magento/pull/393